### PR TITLE
Create react app in initial repository

### PR DESCRIPTION
**Pull request by Jessica-Erickson**

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
+    {
+      "src": "favicon.ico",
```
Comment (Salamandastron1) [inline public/manifest.json]: great icon choice

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
```
Comment (Salamandastron1) [inline src/App.js]: Maybe try angular?

File: src/serviceWorker.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    // The URL constructor is available in all browsers that support SW.
+    const publicUrl = new URL(process.env.PUBLIC_URL, window.location.href);
+    if (publicUrl.origin !== window.location.origin) {
+      // Our service worker won't work if PUBLIC_URL is on a different origin
+      // from what our page is served on. This might happen if a CDN is used to
+      // serve assets; see https://github.com/facebook/create-react-app/issues/2374
+      return;
+    }
+
+    window.addEventListener('load', () => {
```
Comment (Salamandastron1) [inline src/serviceWorker.js]: ES5 nice

**Comment by bghalami** [comment]
As far as  I can tell this looks good.

**Pull request merged**